sql/moz-fx-data-shared-prod/monitoring_derived/shredder_rows_deleted_v1/query.py

```diff
@@ -0,0 +1,138 @@
+"""Monitor the number of rows deleted by shredder."""
+import datetime
+from argparse import ArgumentParser
+from pathlib import Path
+from multiprocessing.pool import ThreadPool
+
+import yaml
+from google.api_core.exceptions import BadRequest
+from google.cloud import bigquery
+
+from bigquery_etl.util import standard_args
+
+JOB_INFO_QUERY = """
+WITH shredder AS (
+  SELECT DISTINCT
+    job_id AS full_job_id,
+    SPLIT(job_id, ":")[OFFSET(0)] AS project_id,
+    SPLIT(job_id, ".")[OFFSET(1)] AS job_id,
+  FROM
+    `moz-fx-data-shredder.shredder_state.shredder_state`
+),
+jobs AS (
+  -- https://cloud.google.com/bigquery/docs/information-schema-jobs
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shredder.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
+  UNION ALL
+  SELECT
+    *
+  FROM
+    `moz-fx-data-bq-batch-prod.region-us.INFORMATION_SCHEMA.JOBS_BY_PROJECT`
+),
+successful_jobs AS (
+  SELECT
+    *,
+  FROM
+    jobs
+  WHERE
+    DATE(creation_time) BETWEEN @end_date - 1 AND @end_date
+    AND DATE(end_time) = @end_date
+    AND state = "DONE"
+    AND error_result IS NULL
+)
+SELECT
+  successful_jobs.end_time,
+  shredder.full_job_id AS job_id,
+  successful_jobs.destination_table.*,
```

Comment:
I'd suggest being explicit here rather than using `*`, both for clarity and to guard against Google changing the output in the future.
```suggestion
  successful_jobs.destination_table.project_id,
  successful_jobs.destination_table.dataset_id,
  successful_jobs.destination_table.table_id,
```